tcl/org.testeditor.tcl.model/src/org/testeditor/tcl/util/TclModelUtil.xtend

```diff
@@ -134,4 +136,16 @@ class TclModelUtil extends TslModelUtil {
 		return null
 	}
 
+	def List<SpecificationStep> getMissingTestSteps(TestCase testCase) {
+		val specSteps = testCase?.specification?.steps
+		val steps = testCase?.steps
+		if( specSteps==null){
+			return Collections.EMPTY_LIST
+		}
+		if( steps==null){
+			return specSteps.toList
+		}
+		return specSteps.filter[!steps.map[it.contents.restoreString].contains(it.contents.restoreString)].toList
```

Comment:
Better than map + contains is to use `exists[predicate]`. The negation would be `forall[predicate]`